app/models/sponsor_seller.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: sponsor_sellers
+#
+#  id          :bigint           not null, primary key
+#  logo_url    :string
+#  name        :string
+#  reward      :string
+#  reward_cost :integer
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  location_id :bigint
+#
+class SponsorSeller < ApplicationRecord
+    validates_presence_of :logo_url, :name, :reward, :reward_cost
```

Comment:
I think :logo_url, :name, :reward you don't need to validate the presence of. Definitely not :logo_url and :reward — but I can see the argument of you wanting to validate name is there